lib/std/os/bits/linux/prctl.zig

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2020 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+
+pub const PR_SET_PDEATHSIG = 1;
```

Comment:
This looks like it should be an enum

Reply:
These definitions stay as close to the `<linux/prctl.h>` header as possible, following the current style of `std.os.bits`. I'm not quite clear what you're picturing with an enum here, care to elaborate?

Reply:
```
pub PR = extern enum(i32) {
  SET_PDEATHSIG = 1,
  GET_PDEATHSIG = 2,

  GET_DUMPABLE = 3,
  SET_DUMPABLE = 4,

  // etc.

  _,
};
```

Reply:
Makes sense. I'm not opposed to that style change, but I think it would be weird to do it just for this PR without a larger movement to refactor `std.os.bits` in the same way. Maybe you could open an issue to discuss?

Reply:
> Maybe you could open an issue to discuss?

I've been slowly going through the std library converting things to exhaustive enums since https://github.com/ziglang/zig/pull/4217#issue-364381196